### ISSUE TITLE
react-native link does not link android project dependencies in version 1.4.0.

### ISSUE THREAD

**Issue by osaario** [react-community/react-native-maps#1192]
react-native link does not link android project dependencies in version 1.4.0. Only ios project is linked. In command line:

```
$ react-native link react-native-maps
rnpm-install info iOS module react-native-maps is already linked 
```

**Comment by guinaldi** [comment]
0.14.0 Version... Same problem here. There is no link for Android and i can't make it work =/

**Comment by jnrepo** [comment]
when I revert back to `react-native-maps: 0.13.1` it works on Android, but not `0.14.0`

**Comment by uxman-sherwani** [comment]
Same issue. react-native link does not link android project dependencies in version 1.4.0. Only ios project is linked.

**Comment by IgorVanian** [comment]
Same here, `react-native: 0.43` `react-native-maps: 0.14`

**Comment by uxman-sherwani** [comment]
I found the source of the issue and submitted a PR https://github.com/airbnb/react-native-maps/pull/1200 for the fix. Until this gets merged, you can use my fork or update your package.json file in your projects node_modules/react-native-maps/package.json.

**Comment by ignivaankushrishi** [comment]
I am facing same issue using `react-native(0.43.2) and "react-native-maps": "^0.14.0"`

**Comment by Kishanjvaghela** [comment]
@uxman-sherwani  its working

**Comment by GayanSandaruwan** [comment]
Not working with version 0.15.
@uxman-sherwani your fix made it... Thanks..

**Comment by alvelig** [comment]
Fixed in airbnb#1313
